spec/systems/upload_keywords_spec.rb

```diff
@@ -10,12 +10,7 @@
 
       visit keywords_path
 
-      within 'form' do
-        fill_in('username', with: user[:username])
-        fill_in('password', with: 'password')
-
-        click_button(I18n.t('auth.login'))
-      end
+      system_login(user.username, 'password')
```

Comment:
as we are using the same `'password'`, how about moving that to the `system_login` method as a default parameter, so we can ignore it when calling `system_login` @gutakk 

```
def system_login(username, password = 'password')
  ....
end

system_login(user.username)
```

Reply:
Fixed in 4735a87b1f5724194213579285c9239d34b56dcd